scenarios/model-catalog/samples/github/azure-openai/gpt-4o/js/azure_ai_inference/getting_started.md

```diff
@@ -0,0 +1,246 @@
+
+
+# Getting Started
+
+- [Create a personal access token](#create-a-personal-access-token)
+- [Install dependencies](#install-depedencies)
+- [Set environment variables](#set-environment-variables)
+- [Run a basic code sample](#run-a-basic-code-sample)
+- [Explore more code snippets](#explore-more-samples)
+
+## 1. Create a personal access token
+
+You'll need to **[create a token](https://github.com/settings/tokens)** to enable free API access. The token will be sent to a Microsoft service but does not need any permissions.
+
+## 2. Install dependencies
+
+Install Azure AI Inferencing package using the following command:
+
+```
+WORK IN PROGRESS
+```
+
+## 3. Set environment variables
+Update or create an environment variable to set your token as the key for the client code.
+
+```bash
+export TOKEN="<your-key-goes-here>"
```

Comment:
export TOKEN="<your-key-goes-here>" - should say "<your-github-token-goes-here>"